src/main/scala/SbtWebBuildBase.scala

```diff
@@ -26,8 +26,11 @@ object SbtWebBase extends AutoPlugin {
     sbtPlugin := true,
     scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings"),
 
-    ScriptedPlugin.scriptedLaunchOpts <+= version apply { v => s"-Dproject.version=$v" },
-    ScriptedPlugin.scriptedLaunchOpts += "-XX:MaxPermSize=256m",
+    crossSbtVersions := Seq("0.13.16", "1.0.0-RC3"),
```

Comment:
You probably need to pick Scala version according to the sbt versions being used:

```scala
scalaVersion := (CrossVersion partialVersion (sbtVersion in pluginCrossBuild).value match {
  case Some((0, 13)) => "2.10.6"
  case Some((1, _)) => "2.12.3"
  case _ => sys error s"Unhandled sbt version ${(sbtVersion in pluginCrossBuild).value}"
})
```